src/_guides/language/language-tour.md

```diff
@@ -286,7 +286,7 @@ Any variable can have a constant value.
 // const [] creates an empty, immutable list (EIL).
 var foo = const []; // foo is currently an EIL.
 final bar = const []; // bar will always be an EIL.
-const baz = const []; // baz is a compile-time constant EIL.
+const baz = []; // baz is a compile-time constant EIL.
```

Comment:
We need to have a note here about being able to omit the `const`.

Reply:
I've reworked the examples and prose in the area of the tour. PTAL